### PR TITLE
Max dev order completion

### PR DESCRIPTION
**Pull request by McSobkov**
Wanted to open this up for PR, but I still need to do some validations and mocha testing on it. The code in here is a bit trickier than the last, so I would appreciate feedback while working on those fixes!

### CODE REVIEW

File: dronuts-backend/src/endpoints/order_routes.js [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
             new_order.validate();
-
             collection.insertOne(new_order, function() {
                 client.close();
             });
+            persistOrderStatus(orderId);
         } catch (error) {
-            console.log(error);
-            return false;
+            throw err;
```
Comment (phi-h-nguyen) [inline dronuts-backend/src/endpoints/order_routes.js:20]: Is there a way we can keep a log of errors so we can keep track of things that go wrong in the backend?

**Review by phi-h-nguyen — COMMENTED** [review]
Left a comment about logging errors, but other than that lgtm!

File: dronuts-backend/src/endpoints/order_routes.js (hunk 1 above)
Comment (McSobkov) [reply dronuts-backend/src/endpoints/order_routes.js:20]: Actually, yeah. I should keep the logging in.

**Review by SeanEZhou — APPROVED** [review]
looks good!

**Review by khushi02 — APPROVED** [review]

**Pull request merged**